js/main.js

```diff
@@ -0,0 +1,18 @@
+const $inputImage = document.querySelector('.input-image');
+const $entryImage = document.querySelector('.entry-image');
+
+$inputImage.addEventListener('input', function (event) {
+  $entryImage.setAttribute('src', $inputImage.value);
+});
+
+const $form = document.querySelector('#new-entry');
+
+$form.addEventListener('submit', function (event) {
+  event.preventDefault();
+  const newEntry = {};
+  newEntry.entryId = data.nextEntryId;
```

Comment:
You should have 3 other properties inside this object as well. We should be storing all of the form data inside this object, along with the id